### PR TITLE
162 Adding LB Access Logs

### PR DESCRIPTION
**Pull request by charles-nava**
## Ticket
[template-infra #162](https://github.com/orgs/navapbc/projects/4/views/1?filterQuery=162&pane=issue&itemId=13741361)

## Changes

> Adds access logs
> Adds s3 for access logs
> Associated IAM for access logs to s3
> S3 module for easy compliance with relevant checks

## Context for reviewers

> New pull request after reverting to platform test repo

## Testing

> Deployed successfully to platform AWS
> ALB logs show up as expected 
> Screenshot 2023-08-08 at 1 27 56 PM
> Tears down successfully (see End-to-End tests)

### CODE REVIEW

File: infra/test/infra_test.go
Comment (lorenyu) [inline infra/test/infra_test.go:1]: @charles-nava don't forget to include the changes in this file in https://github.com/navapbc/template-infra/pull/362

File: infra/app/service/dev.tfvars [hunk 1]
```diff
@@ -2,14 +2,3 @@ environment_name = "dev"
 tfstate_bucket   = "platform-test-430004246987-us-east-1-tf"
 tfstate_key      = "infra/app/service/dev.tfstate"
 region           = "us-east-1"
-db_vars = {
```
[the remark this answers is not in the document]
Comment (charles-nava) [reply infra/app/service/dev.tfvars:5]: It was not intentional or at the least I don't remember doing it

[the remark this answers is not in the document]
Comment (lorenyu) [reply infra/app/service/dev.tfvars]: Actually this is my bad, I realize this db_vars isn't actually used and is probably leftover from something I did a while ago. I'll go ahead and delete it on the main branch. Sorry for my confusion.

[the remark this answers is not in the document]
Comment (lorenyu) [reply infra/app/service/dev.tfvars]: image

**Comment by charles-nava** [comment]
Merged in template-infra

**Pull request closed without merging**